src/main/kotlin/org/neo4j/graphql/Translator.kt

```diff
@@ -397,7 +459,26 @@ object SchemaBuilder {
         """
         typeDefinitionRegistry.merge(schemaParser.parse(directivesSdl))
 
-        val augmentations = typeDefinitionRegistry.types().values.filterIsInstance<ObjectTypeDefinition>().map { augmentedSchema(ctx, it) }
+        val objectTypeDefinitions = typeDefinitionRegistry.types().values.filterIsInstance<ObjectTypeDefinition>()
+        val nodeAugmentations = objectTypeDefinitions.map { augmentedSchema(ctx, it) }
+        val relAugmentations = objectTypeDefinitions.flatMap { source ->
+            val fields = source.fieldDefinitions.filter { !it.type.isScalar() && it.directives.isEmpty() }
+            if (fields.isEmpty()) {
```

Comment:
we don't need the IF 
we can just use fields.mapNotNull which will also result in an empty list

Reply:
cool thanks!